modules/store.js

```diff
@@ -0,0 +1,33 @@
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  // add books to localStorage
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  // remove a book from local storage
+  static removeBook(title) {
+    const books = Store.getBooks();
+
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```

Comment:
- [ ] Your app does not really clear the local storage on removing the books. When I remove books and reload all the books still come back on the page. Kindly fix this.

  book is added | Book is removed | page is refreshed
  :---:|:---:|:---:
  ![aw-1](https://user-images.githubusercontent.com/54848363/183485001-3111eedc-4613-4e3b-9295-b486e2cad99a.jpg) | ![aw-2](https://user-images.githubusercontent.com/54848363/183484992-97f1e401-f284-48bf-9138-5ceb7971aea7.jpg) | ![aw-1](https://user-images.githubusercontent.com/54848363/183485001-3111eedc-4613-4e3b-9295-b486e2cad99a.jpg)